app/views/foreman_webhooks/webhook_templates/empty_payload.erb

```diff
@@ -0,0 +1,6 @@
+<%#
+name: Empty Payload
+description: Useful for Shellhooks which only takes arguments via HTTP parameters.
+snippet: false
+model: WebhookTemplate
+-%>
```

Comment:
Empty payload is useful when using Shellhooks and passing arguments via HTTP headers, e.g.:

https://github.com/theforeman/smart_proxy_shellhooks/blob/master/examples/ansible-tower-cli

Reply:
To be more safe, I'd go with:

<%= payload({}, with_defaults: false) -%>